doc/src/stdlib/base.md

```diff
@@ -134,6 +168,7 @@ Base.method_exists
 Core.applicable
 Core.invoke
 Base.invokelatest
+new
```

Comment:
Does this not need a prefix? Maybe the latest docs haven't been updated yet, but it isn't showing up.

Reply:
Hmm... It worked locally, and Documenter would have errored on CI unless it found the reference.

Reply:
This is more of a keyword since there's no actual identifier with this name.